cardano-lib/generic-log-config.nix

```diff
@@ -7,208 +7,80 @@
   # These metrics can be directed to the logs or monitoring backends.
   TurnOnLogMetrics = true;
 
-  # Global logging severity filter. Messages must have at least this severity to
-  # pass. Typical values would be Warning, Notice, Info or Debug.
-  minSeverity = "Info";
-
-  # Log items can be rendered with more or less verbose detail.
-  # The verbosity can be: MinimalVerbosity, NormalVerbosity
-  TracingVerbosity = "NormalVerbosity";
-
-  # The system supports a number of backends for logging and monitoring.
-  # This setting lists the the backends that will be available to use in the
-  # configuration below. The logging backend is called Katip. Also enable the EKG
-  # backend if you want to use the EKG or Prometheus monitoring interfaces.
-  setupBackends = [
-    "KatipBK"
-    # EKGViewBK
+  # Use the modern tracing system
+  UseTraceDispatcher = true;
+
+  # Set severities for the tracing namespace
+  TraceOptionSeverity = [
+
+    # Globally, i.e. default:
+    { ns = "";                      severity = "Notice"; };
```

Comment:
How can this `;` at the end be parsed? :open_mouth:

Reply:
Oh, wow -- thank you for catching this @michalrus !